macros/users_table.sql

```diff
@@ -72,3 +72,52 @@ from
 where
     rn = 1
 {% endmacro %}
+
+{% macro databricks__users_table(user_cols, user_keys, user_types) %}
+
+{# Remove down to major version for Snowflake columns, drop 2 last _X values #}
+{%- set user_cols_clean = [] -%}
+{%- for ind in range(user_cols|length) -%}
+    {% do user_cols_clean.append('_'.join(user_cols[ind].split('_')[:-2])) -%} 
+{%- endfor -%}
+
+{# Replace keys with snake_case where needed #}
+{% set re = modules.re %}
+{% set camel_string = '(?<!^)(?=[A-Z])'%}
+{%- set user_keys_clean = [] -%}
+{%- for ind1 in range(user_keys|length) -%}
+    {%- set user_key_clean = [] -%}
+    {%- for ind2 in range(user_keys[ind1]|length) -%}
+        {% do user_key_clean.append(re.sub(camel_string, '_', user_keys[ind1][ind2]).lower()) -%} 
+    {%- endfor -%}
+    {% do user_keys_clean.append(user_key_clean) -%} 
+{%- endfor -%}
+
+
+with users_ordered as (
```

Comment:
Are they actually ordered? I don't see an ORDER BY statement or is that not necessarily what you mean by ordered?

Reply:
I meant "ordered" in that there is a key that you can now order on, think it should have a different name?